Tests/configure_and_test_integration_instances.py

```diff
@@ -693,9 +737,10 @@ def main():
     # [conf.json](../conf.json) for examples
     brand_new_integrations = []
     testing_server = servers[0]  # test integration instances only on a single server
+    testing_client = demisto_client.configure(base_url=testing_server, username=username, password=password,
```

Comment:
there was a reason this was inside the loop.
for long test list, the session will be terminated by Demisto